sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.2.2 (2021-01-12)
```

Comment:
Just make sure, there are no ongoing changes on the master repo. The patch release should just be the dependency version bump from the previous release.